Neos.Flow/Classes/Mvc/Controller/AbstractController.php

```diff
@@ -364,7 +364,7 @@ protected function mapRequestArgumentsToControllerArguments()
     {
         foreach ($this->arguments as $argument) {
             $argumentName = $argument->getName();
-            if ($this->request->hasArgument($argumentName)) {
+            if ($this->request->hasArgument($argumentName) && !empty($this->request->getArgument($argumentName))) {
```

Comment:
If we really only want to avoid setting `null` we should better explicitly test against null here. `empty` will also avoid empty string, zero values (also `"0"`), false and empty array. That's not what we want.